api/src/main/java/org/apache/iceberg/types/ReassignIds.java

```diff
@@ -45,7 +51,9 @@ public Type schema(Schema schema, Supplier<Type> future) {
   }
 
   private int id(Types.StructType sourceStruct, String name) {
-    Types.NestedField sourceField = sourceStruct.field(name);
+    Types.NestedField sourceField =
+        caseSensitive ? sourceStruct.field(name) : sourceStruct.caseInsensitiveField(name);
```

Comment:
any reason why this is a different style than the version on 97?

Reply:
I think the line on 97 is slightly longer because it uses `field.name()` instead of `name` when passing argument to the `sourceStruct` functions.

So it's the formatter due to line lengths if I'm not mistaken. Sometimes I find you can refactor in a different way (like matching the style from line 97) and the formatter will accept it.

Reply:
ah, i missed that difference. I thought they were identical